about.html

```diff
@@ -0,0 +1,129 @@
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </header>
+<!--HERO-->
+        <section class="hero">
+            <p class="quote">
+                “If You Think Math is Hard Try Web Design”
+            </p>
+            <h1 class="first-heading">
+                DesignX Company<br>
+                Web Design<br>
+                2023
+            </h1>
+            <div class="first-para">
+                <p>
+                    The main programs of a web design conference 
+                    typically include keynote speeches, panel discussions, 
+                    workshops, and networking sessions. These programs aim to provide attendees with insights, knowledge, and practical 
+                    skills related to web design and development.
+                </p>
+            </div>
+            <p class="second-heading">
+                Please contact us per Email for any 
+                further questions about DesignX company.
+            </p>
+            <p class="second-para">
+                <a href="/">
+                DesignX2023@gmail.com
+                </a>
+            </p>
+        </section>
+    </div>
+<!--MAIN PROGRAM-->
+    <section class="main-program">
+            <h2>Creative Commons DesignX Company 2023 Logo</h2>
+            <hr>
+            <p>
+                The logo of DesignX company 2023 was decided
+                through the logo competition from 8, June to 7, July.
+            </p>
+            <img src="images/dark logo.png" alt="logo">
+    </section>
+    <hr class="hr">
+<!--FEATURED-->
+    <section class="featured-container">
+        <h2>See the past DesignX Confernecs</h2>
+            <hr>
+            <p>
+                Take a look at the past two DesignX Confernces
+                which took place in Sadaf Tower and Hoding Group.
+            </p>
+            <div class="ci-container">
+                <div class="ci-one">
+                    <div class="ci">
+                        <h3>2015</h3>
+                        <p>DesignX Conference 2015 in Sadaf Tower</p>
+                    </div>
+                </div>
+                <div class="ci-two">
+                    <div class="ci">
+                        <h3>2019</h3>
+                        <p>DesignX Conference 2019 in Sadaf Tower</p>
+                    </div>
+                </div>
+            </div>
+    </section>
+<!--SPONSOR-->
+    <section class="sponsor-container">
+        <div class="sponsor-heading">
+            <h2>Partner</h2>
+            <hr>
+        </div>
+        <div class="sponsor-images">
+            <img src="images/sponsor-1.svg" alt="sponsor-1">
+            <img src="images/sponsor-2.svg" alt="sponsor-2">
+            <img src="images/sponsor-3.svg" alt="sponsor-3">
+            <img src="images/sponsor-4.svg" alt="sponsor-4">
+            <img src="images/sponsor-5.svg" alt="sponsor-5">
```

Comment:
- Good job done so far 👍 👍, To help improve the semantics of your page, I suggest that you wrap this section in a `ul` tag and wrap all individual images in a corresponding `li` tag. Please fix this 👍 👍